composer.json

```diff
@@ -1,5 +1,20 @@
 {
     "require": {
-        "google/apiclient": "^2.5"
+        "php": ">=7.2.5",
+        "google/apiclient": "^2.5",
+        "google/apiclient-services": "v0.138"
```

Comment:
I've set a fixed version of the client-services to prevent any problems when running composer update. `google/apiclient` requires no specific version of it, so it always uses the latest (which could include any breaking changes)